Sources/ShortcutFoundation/Core/Device/Biometrics/DeviceAuthentication.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  DeviceAuthentication.swift
+//  
+//
+//  Created by Swathi on 2022-02-02.
+//
+
+import LocalAuthentication
+
+//Current state if authentication
+public enum DeviceAuthAction: Equatable {
+    case authenticated
+    case notAuthenticated
+    case error(BiometricError)
+}
+
+//Error states while authentication
+public enum BiometricError: Swift.Error {
```

Comment:
Any particular reason to add the Swift namespace? Since we do not have any Error, the compiler will understand that is the built-in type.

Reply:
Removed "Swift" keyword